build.gradle

```diff
@@ -78,7 +78,7 @@ ext.library = [
     error_prone_annotations: "com.google.errorprone:error_prone_annotations:2.0.15",
     findbugs_annotations: "com.github.stephenc.findbugs:findbugs-annotations:1.3.9-1",
     findbugs_jsr305: "com.google.code.findbugs:jsr305:3.0.1",
-    gax_grpc: "com.google.api:gax-grpc:0.20.0",
+    gax: "com.google.api:gax:1.19.0",
```

Comment:
Actually do we even need to specify gax version here? Only reference to gax I see in the beam repo is in a test which I think can be removed:
https://github.com/apache/beam/search?utf8=%E2%9C%93&q=gax&type=

Reply:
Oh i see you added a reference to this below. Ignore this.